cmake/UnitTests.cmake

```diff
@@ -1571,9 +1568,6 @@ list(APPEND DGPU_LEVEL0_FAILED_TESTS "Unit_hipPeekAtLastError_Positive_Threaded"
 list(APPEND DGPU_LEVEL0_FAILED_TESTS "hipDynamicShared") # SEGFAULT
 
 # iGPU Level Zero Unit Test Failures
-list(APPEND IGPU_LEVEL0_FAILED_TESTS "sycl_chip_interop_usm") # Subprocess aborted
-list(APPEND IGPU_LEVEL0_FAILED_TESTS "hip_sycl_interop") # Subprocess aborted
-list(APPEND IGPU_LEVEL0_FAILED_TESTS "hip_sycl_interop_no_buffers") # Subprocess aborted
 list(APPEND IGPU_LEVEL0_FAILED_TESTS "cuda-template") # SEGFAULT
```

Comment:
`cuda-template` is now first in list. Anything to do with it not getting excluded?